CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ BUG FIXES:
  * api: Added missing devices block to AllocatedTaskResources [[GH-10064](https://github.com/hashicorp/nomad/pull/10064)]
  * cli: Fixed a bug where non-int proxy port would panic CLI [[GH-10072](https://github.com/hashicorp/nomad/issues/10072)]
  * cli: Fixed a bug where `nomad operator debug` incorrectly parsed https Consul API URLs. [[GH-10082](https://github.com/hashicorp/nomad/pull/10082)]
+ * cli: Update defaults for `nomad operator debug` flags `-interval` and `-server-id` to match common usage. [GH-10121]
```

Comment:
`make changelogfmt` will take this tag and automatically do the right thing for the URL.

Reply:
Fixed.

Reply:
Not quite, this should now be reading `[[GH-10121](https://github.com/hashicorp/nomad/pull/10121)]

Reply:
Not sure what happened during the rebase to main, but this is back in place now.